### PR TITLE
Feature/10 dse has a slack home generating package

### LINKED ISSUE #10: DSE has a Slack home screen with fake data

### PR DESCRIPTION
**Pull request by Startouf**
resolves partially #10 

A first PR to lay out the basic blocs, and validate the way to implement block kit / test it using go. Of course this is far from being the finished home :P 

[Preview](https://api.slack.com/tools/block-kit-builder?mode=appHome&view=%7B%22type%22%3A%22home%22%2C%22blocks%22%3A%5B%7B%22type%22%3A%22section%22%2C%22text%22%3A%7B%22type%22%3A%22mrkdwn%22%2C%22text%22%3A%22*Don%27t%20Slack%20Evil%20Performance*%22%7D%2C%22accessory%22%3A%7B%22type%22%3A%22button%22%2C%22text%22%3A%7B%22type%22%3A%22plain_text%22%2C%22text%22%3A%22Manage%20App%20Settings%22%2C%22emoji%22%3Atrue%7D%2C%22value%22%3A%22app_settings%22%7D%7D%2C%7B%22type%22%3A%22section%22%2C%22text%22%3A%7B%22type%22%3A%22plain_text%22%2C%22text%22%3A%22%20%3Awave%3A%20Hello%20Cyril%20%C2%B7%20find%20your%20DSE%20stats%20below%22%2C%22emoji%22%3Atrue%7D%7D%2C%7B%22type%22%3A%22divider%22%7D%5D%7D)

### CODE REVIEW

File: apphome/apphome.go [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+		hellloSection,
+		divSection,
+	)
+	message.Msg.Type = "home"
+
+	return message
+}
+
+func translateUserIdToUserName(userId string) string {
+	// TODO
```
Comment (gjgd) [inline apphome/apphome.go]: Can we create a ticket for that?

File: apphome/apphome.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (gjgd) [inline apphome/apphome.json]: where is this used?

**Review by gjgd — COMMENTED** [review]
Can you link a pic of what the apphome looks like?

File: apphome/apphome.json (hunk 2 above)
Comment (Startouf) [reply apphome/apphome.json]: AH sorry it should not be part of this PR, but the next one ^^"

File: apphome/apphome.go (hunk 1 above)
Comment (Startouf) [reply apphome/apphome.go]: Sure, https://github.com/gjgd/dont-slack-evil/issues/16

**Review by gjgd — APPROVED** [review]
LGTM 👍

**Review by mourtisma — APPROVED** [review]

**Pull request merged**